src/app/app.component.ts

```diff
@@ -14,6 +14,8 @@ import { Notifications } from '../providers/notifications';
   templateUrl: './app.html'
 })
 export class StockpileApp {
+  // Need to ViewChild Nav instead of using NavController because this is
```

Comment:
Do you have a preference of `//` comments to `/* */` comments? I think the latter style of comment is better for multi-line comments, so it would be a sensible standard to adopt across the whole codebase, even for functions with one-line descriptions.